STAKE-VALIDATOR.md

```diff
@@ -175,10 +175,9 @@ data ScriptContext =
 Within the `TxInfo` type, note the importance of the `txInfoWdrl :: Map StakingCredential Integer` field.
 
 This field encapsulates a Map where each `StakingCredential` serves as a key, paired with its corresponding withdrawal amount as the associated value.
-> Note: The `txInfoWdrl` contains all the staking credentials attempting to withdraw rewards from their staking accounts.
-> Note: as a side note , there is no such thing as Map type, the underlying type is just a builtin list of builtin pairs [^1]
+> Note: The `txInfoWdrl` contains all the staking credentials attempting to withdraw rewards from their staking accounts. Also keep in mind that the Map type in Plutus Core is implemented as a list of builtin pairs [^1].
```

Comment:
IMHO this tweak is more precise, since the Plutus Core specification (both old and new version) explicitly talks about the Plutus `Map` type.